Buildings/Fluid/DXSystems/Cooling/AirSource/Validation/Data/SingleSpeedHeating.mo

```diff
@@ -0,0 +1,44 @@
+within Buildings.Fluid.DXSystems.Cooling.AirSource.Validation.Data;
+record SingleSpeedHeating
+  "Data record for DX heating coil in validation models"
+  extends
+    Buildings.Fluid.DXSystems.Heating.AirSource.Data.Generic.Coil(
+    sta={
+        Buildings.Fluid.DXSystems.Cooling.AirSource.Data.Generic.BaseClasses.Stage(
```

Comment:
This is a data record for cooling. It must be changed to heating and there should be no parameter such as `tWet` and `gamma`